### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by Gilwyad** [microsoft/vscode-htmlhint#61]
Installed 0.6.0 on VS Code 1.35 and it does nothing, invalid HTML files are not highlighted.

**Comment by kanlukasz** [comment]
Can you share a file with us that is not highlighted?
Have you tried version 1.40?
Have you tried with other extensions turned off?

**Comment by Gilwyad** [comment]
Here's one:

    <html>
    <body>
        <ol>
            <li>

        </ol>
        </li>
    </body>
    </html>

Nothing is highlighted in VS Code. Tried with no htmlhint cli installed, with 0.11.0 installed globally, on VS Code 1.40.1 , all other extensions disabled.

**Comment by kanlukasz** [comment]
I tried that right now with 1.40.2 - everything works fine, look at my screenshot:

![obraz](https://user-images.githubusercontent.com/30685349/69663334-57a03280-1086-11ea-8a5d-360c91eec0f0.png)

**Have you saved the file? Please note that the plugin cannot work (from what I see) if the file does not exist physically on your disk**

_

**Comment by Gilwyad** [comment]
I tried it on a Windows machine and indeed it works. However I am using Ubuntu 18 and that's where it doesn't work. How can I debug this? There should be a way that I can see the extensions error messages.

**Comment by kanlukasz** [comment]
I am sorry to hear it. I'm not the author of the plugin, so I don't know how to help you more. What's worse, from what I see, the author has not been fixing problems for some time :(

**Comment by Gilwyad** [comment]
I was able to track down the problem: if I have this configuration set:

    "htmlhint.options": {
        "attr-lowercase": false
    },
  
then the extension doesn't work. But if I remove it (and restart) then it does work!

And I was testing with lower case tags, e.g. <h5></h4> so it should show the error regardless of case.

**Comment by tahaerden** [comment]
> I was able to track down the problem: if I have this configuration set:
> 
> ```
> "htmlhint.options": {
>     "attr-lowercase": false
> },
> ```
> 
> then the extension doesn't work. But if I remove it (and restart) then it does work!
> 
> And I was testing with lower case tags, e.g. `<h5></h4>` so it should show the error regardless of case.

Yes I just installed the extension and noticed the htmlhint.options override all options and not only the ones inside this object.

**Comment by vatro** [comment]
> > I was able to track down the problem: if I have this configuration set:
> > ```
> > "htmlhint.options": {
> >     "attr-lowercase": false
> > },
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > then the extension doesn't work. But if I remove it (and restart) then it does work!
> > And I was testing with lower case tags, e.g. `<h5></h4>` so it should show the error regardless of case.
> 
> Yes I just installed the extension and noticed the htmlhint.options override all options and not only the ones inside this object.

I've just noticed that too.
